src/Symfony/Bundle/FrameworkBundle/Command/CachePoolSetCommand.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Sets a cache pool key value.
+ *
+ * @author Antoine Makdessi <amakdessi@me.com>
+ */
+final class CachePoolSetCommand extends Command
+{
+    protected static $defaultName = 'cache:pool:set';
+
+    private $pools;
+
+    public function __construct(array $pools)
+    {
+        parent::__construct();
+
+        $this->pools = $pools;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Sets a cache pool key value.')
+            ->addArgument('pool', InputArgument::REQUIRED, 'The cache pool to which to set a key')
+            ->addArgument('key', InputArgument::REQUIRED, 'The cache key to set a value on')
+            ->addArgument('value', InputArgument::REQUIRED, 'The value to set in the cache pool key')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> sets a value on a key on a given cache pool.
+
+    %command.full_name% <pool> <key> <value>
+EOF
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $pool = $input->getArgument('pool');
+        if (!\array_key_exists($pool, $this->pools)) {
+            $io->error('Error loading the cache pool.');
+
+            return Command::FAILURE;
+        }
+
+        $cache = $this->pools[$pool];
+
+        $key = $input->getArgument('key');
+        $value = $input->getArgument('value');
+
+        $cacheItem = $cache->getItem($key);
+        $cacheItem->set($value);
+        $isSaved = $cache->save($cacheItem);
+
+        if ($isSaved) {
+            $io->info(sprintf('Cache item value was successfully saved in "%s" "%s".', $pool, $key));
```

Comment:
I think success is appropriate for this one. As it updates a value, a strong feedback is welcomed.